_clanky/geolokační-data-mobilních-operátorů-principy-příklady-otázky.md

```diff
@@ -72,19 +72,19 @@ Případně je možné stejná data kategorizovat dle typologie osob, tedy zda j
 
 Druhým typem dat jsou data o dojížďce, tedy samotné vztahy v území. Tento typ dat je lepší pro znázorňování nejvýznamnějších dopravních proudů. Je samozřejmě nutné si stanovit parametry, které pohyby chceme sledovat, v jakém rozsahu, po jak dlouhou dobu apod. Dodaný výstup tak může vypadat například takto:
 
-| start_cas | cil_cas | start_level | start_kod | cil_level | cil_kod | pocet_kalibrovano |
-|-----------|---------|-------------|-----------|-----------|---------|-------------------|
-| 0         | 1       | 1           | 608505    | 1         | 608505  | 7.87              |
-| 0         | 1       | 1           | 610003    | 1         | 610003  | 8.27              |
-| 0         | 1       | 1           | 610089    | 1         | 610089  | 2.55              |
-| 0         | 1       | 1           | 610186    | 1         | 610186  | 2.55              |
+| start_cas | cil_cas | start_level | start_kod | cil_level | cil_kod | pocet_kalibrovano |
+| --------- | ------- | ----------- | --------- | --------- | ------- | ----------------- |
+| 0         | 1       | 1           | 608505    | 1         | 608505  | 7.87              |
+| 0         | 1       | 1           | 610003    | 1         | 610003  | 8.27              |
+| 0         | 1       | 1           | 610089    | 1         | 610089  | 2.55              |
+| 0         | 1       | 1           | 610186    | 1         | 610186  | 2.55              |
 | 0         | 1       | 1           | 610208    | 1         | 610208  | 2.55              |
 
 Kde položka “start_cas” a cil_cas” nabývá hodnot 0-23, “start_level” a “cil_level” jsou úrovně administrativní jednotek (katastr, obec, so orp, kraj) “start_kod” a “cil_kod” zase konkrétní kódy administrativních celků, “pocet_kalibrovano” je počet osob přepravených mezi danými územími.
 
 ## Příklady - co s daty dělat?
 
-V této ukázce budou vizualizovány základní přepravní proudy ve městě Brně prostřednictvím open-source geografického nástroje [QGIS][QGIS]. Cílem bude zobrazit hlavní přepravní proudy mezi katastrálními územími (48 jednotek) a zjistit tak, které katastrální vazby jsou v Brně nejsilnější. Níže v mapě vidíte, k jakému výsledku chceme dojít.
+V této ukázce budou vizualizovány základní přepravní proudy ve městě Brně prostřednictvím open-source geografického nástroje [QGIS](https://qgis.org/ "QGIS"). Cílem bude zobrazit hlavní přepravní proudy mezi katastrálními územími (48 jednotek) a zjistit tak, které katastrální vazby jsou v Brně nejsilnější. Níže v mapě vidíte, k jakému výsledku chceme dojít.
```

Comment:
open source bych napsal bez pomlčky
co je to katastrální vazba? není to spíš vazba mezi KÚ?